pkg/vm/engine/tae/catalog/schema.go

```diff
@@ -254,6 +249,19 @@ func (s *Schema) ReadFrom(r io.Reader) (n int64, err error) {
 			return
 		}
 		n += sn
+		var length uint64
```

Comment:
As we talked offline, there is no need to change here. Need a modification in types.EncodeType